migrations/migration_resource.py

```diff
@@ -0,0 +1,78 @@
+"""Migration Resource
+
+Defines a FHIR resource holding data about the latest migration by specializing
+the `fhirclient.Basic` class. A single Basic resource is maintained with the most 
+recently (successfully) run migration revision held in the single Basic.code value.
+"""
+from datetime import datetime
+
+from fhirclient.models.basic import Basic
```

Comment:
minor nitpick: `fhirclient` is pretty awful and would be an ongoing burden to maintain. there may be better alternatives like [fhir.resources](https://pypi.org/project/fhir.resources/)

the biggest issue is that they tightly couple the version of PyPI to the version of FHIR they support. ie `fhirclient==4.0.0` can **only** support FHIR R4